icubam/backoffice/handlers/operational_dashboard.py

```diff
@@ -0,0 +1,151 @@
+import numpy as np
+import tornado.web
+import math
+from collections import defaultdict
+
+from bokeh.embed import components
+from bokeh.plotting import figure
+from bokeh.transform import dodge
+from bokeh.models import ColumnDataSource
+
+from icubam.db.store import to_pandas
+from icubam.backoffice.handlers import base
+
+
+class OperationalDashHandler(base.BaseHandler):
+  ROUTE = '/operational-dashboard'
+
+  @tornado.web.authenticated
+  async def get(self):
+    """Serves a page with a table gathering current bedcount data with some extra information."""
+    arg_region = self.get_query_argument('region', default=None)
+    current_user = self.get_current_user()
+
+    current_region = None
+    current_region_name = "Toutes les régions"
+    if arg_region is not None and arg_region.isdigit():
+      try:
+        current_region = self.db.get_region(int(arg_region))
+        current_region_name = current_region.name
+      except Exception:
+        logging.warning(
+          f"Invalid query argument: region={arg_region} "
+          f"Falling back to all regions."
+        )
```

Comment:
I know I should add tests for this path, but for I haven't managed to make it work (user needed to be monkeypatched, and I was likely missing something).

Reply:
you can just do like the other with mock.patch. Then you can try different requests  with different regions..

Check  backoffice/server_test.py